azure-iot-device/setup.py

```diff
@@ -58,7 +58,12 @@
     install_requires=[
         # Define sub-dependencies due to pip dependency resolution bug
         # https://github.com/pypa/pip/issues/988
-        "urllib3>1.21.1,<1.25",
+        # ---requests dependencies---
+        # requests 2.22+ does not support 1.25.0 or 1.251 (https://github.com/psf/requests/pull/5092)
+        # requests 2.22+ is not compatible with python34
+        "urllib3>1.21.1,<1.26,!=1.25.0,!=1.25.1;python_version=='2.7'",
```

Comment:
>"urllib3>1.21.1,<1.26,!=1.25.0,!=1.25.1;python_version=='2.7'", [](start = 8, length = 63)

Is there a way to collapse your 2.7 and 3.5+ lines together and just say python_version != '3.4'?

Reply:
That's a good idea - I didn't think of that. I tried everything except using a !=.